statsdaemon.go

```diff
@@ -289,12 +289,12 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 
 		switch {
 		case hasChanged:
-			fmt.Fprintf(buffer, "%s %d %d\n", bucket, currentValue, now)
+			fmt.Fprintf(buffer, "%s %.0f %d\n", bucket, currentValue, now)
```

Comment:
Actually looks like it's a moot point -- we can format them all with `strconv.FormatFloat(n, 'f', -1, 64)` which includes the minimum number of decimal places to represent it exactly (and appears to omit the decimal point entirely for integers).